server/models.js

```diff
@@ -0,0 +1,35 @@
+const mysql = require('mysql');
+
+const connection = mysql.createConnection({
+  host: 'localhost',
+  user: 'root',
+  password: 'Password',
```

Comment:
I think it is best practice to not upload your password to github, so what you can do instead is to have a config file which you import. Git ignore the config file. Use variables in place of the actual passwords